psd-web/app/controllers/investigations/coronavirus_related_controller.rb

```diff
@@ -0,0 +1,20 @@
+module Investigations
+  class CoronavirusRelatedController < ApplicationController
+    def show
+      @investigation = Investigation.find_by(pretty_id: params.require(:investigation_pretty_id)).decorate
```

Comment:
I think we should do this:

```suggestion
      @investigation = Investigation.find_by!(pretty_id: params.require(:investigation_pretty_id)).decorate
```

so that it throws a 404 page rather than a 500 error if the investigation isn’t found.